napari/layers/graph/graph.py

```diff
@@ -399,10 +401,27 @@ def add(
         coords : sequence of indices to add point at
         indices : optional indices of the newly inserted nodes.
         """
+        self.events.data(
+        value=self.data,
+        action=ActionType.ADDING,
+        data_indices=tuple(
+                self.selected_data,
```

Comment:
I don't know if this should be `self.selected_data` or just `-1`?  In the points layer when we add we just use `-1` as the `data_indices`, so I think we should do the same here (even though probably it wouldn't change the behaviour since adding a point also selects it)